src/SearchResults.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import moment from 'moment'
+
+class SearchResults extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { isRowSelected: false, selectedRow: null };
+    }
+
```

Comment:
if you read HW task 14 carefully, you'll see that many rows can be selected at the same time.  So you need a data structure(s) that can track that.  A single scalar variable (e.g. a single number) isn't going to be enough to model the case that the rows 1, 2, 5, 17, and 19 are selected!

Once you decide on the correct state to use, it will be easier to write everything else.